src/bin/vhost_user_net.rs

```diff
@@ -433,6 +433,10 @@ impl VhostUserBackend for VhostUserNetBackend {
             | VhostUserVirtioFeatures::PROTOCOL_FEATURES.bits()
     }
 
+    fn protocol_features(&self) -> VhostUserProtocolFeatures {
```

Comment:
Yes that makes a lot of sense, I had forgot about that! Thanks for fixing it, we're gonna need it once we want to support complex features required by `vhost-user-fs`!